aws/sns/sns.go

```diff
@@ -0,0 +1,74 @@
+// Package sns is for messaging with AWS SNS.
+package sns
+
+import (
+	"context"
+	"os"
+
+	"github.com/pkg/errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/retry"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/sns"
+)
+
+type SNS struct {
+	client *sns.Client
```

Comment:
I assume the client is here to protect the wrapped sns.Client functions from being called externally (that is they are hidden).